scripts/globals/goblinfootprint.lua

```diff
@@ -0,0 +1,218 @@
+-----------------------------------
+-- Goblin Footprint NPCs
+-----------------------------------
+require("scripts/globals/quests")
+require("scripts/globals/missions")
+require("scripts/globals/zone")
+
+dsp = dsp or {}
+dsp.goblinfootprint = dsp.goblinfootprint or {}
+
+local gobCS = -- add the goblin cs to this table
+{
+    [dsp.zone.LA_THEINE_PLATEAU]     = 126,
+    [dsp.zone.NORTH_GUSTABERG]       = 247,
+    [dsp.zone.SOUTH_GUSTABERG]       = 905,
+    [dsp.zone.MERIPHATAUD_MOUNTAINS] = 35,
+    [dsp.zone.ROMAEVE]               = 6,
+    [dsp.zone.CASTLE_ZVAHL_KEEP]     = 10,
+    [dsp.zone.BOSTAUNIEUX_OUBLIETTE] = 100,
+    [dsp.zone.MAZE_OF_SHAKHRAMI]     = 67,
+    [dsp.zone.GARLAIGE_CITADEL]      = 61,
+}
+
+local csReq = -- add checks to this table
+{
+    [dsp.zone.LA_THEINE_PLATEAU] =
+    {
+        [1] = function(player) return (player:hasCompletedQuest(BASTOK, dsp.quest.id.bastok.DARK_PUPPET)) end,
+        [2] = function(player) return (player:hasCompletedQuest(WINDURST, dsp.quest.id.windurst.HITTING_THE_MARQUISATE)) end,
+        [3] = function(player) return (player:hasCompletedQuest(WINDURST, dsp.quest.id.windurst.I_CAN_HEAR_A_RAINBOW)) end,
+        [4] = function(player) return (player:hasCompletedQuest(SANDORIA, dsp.quest.id.sandoria.A_TIMELY_VISIT)) end,
+        [5] = function(player) return (player:hasCompletedQuest(SANDORIA, dsp.quest.id.sandoria.A_TIMELY_VISIT)) end,
+        [6] = function(player) return (player:hasCompletedMission(COP, dsp.mission.id.cop.THE_MOTHERCRYSTALS)) end,
+        [7] = function(player) return (player:hasCompletedMission(COP, dsp.mission.id.cop.THREE_PATHS)) end,
+        [8] = function(player) return (player:completeQuest(OTHER_AREAS_LOG, dsp.quest.id.otherAreas.WAKING_THE_BEAST)) end,
+        [9] = function(player) return (player:completeQuest(OTHER_AREAS_LOG, dsp.quest.id.otherAreas.WAKING_THE_BEAST)) end,
```

Comment:
Should these be calling completeQuest?

Reply:
oh crap good catch @cocosolos I musta been asleepy when i wrote this. since it was a massive number of copy and paste lol. Going to change it now